Bricks/util/base64.h

```diff
@@ -74,7 +74,7 @@ struct Impl {
     if (nbits > 0) {
       result.push_back(map[((buf << 6) >> nbits) & 0x3F]);
     }
-    const int num_pads = result_size - result.size();
+    const size_t num_pads = result_size - result.size();
```

Comment:
This conversion was generating a compiler warning, not triggering a data error (and would never do so, because it is logically constrained to the range 0-2) so there's no unit test, but the existing tests pass with the change.